pkg/controller/projectinitialize/projectinitialize_controller.go

```diff
@@ -132,7 +133,23 @@ func (r *ReconcileProjectInitialize) Reconcile(request reconcile.Request) (recon
 				// TODO reverse the project creation?
 			}
 		}
-		logging.Log.Info(fmt.Sprintf("Created new project %s", newProject.Name))
+		if instance.Spec.Git != nil && instance.Spec.GitTemplate != nil {
```

Comment:
Do we want to enforce a template be specified or can we just create a blank repository for teams to fill out?

Reply:
Good idea! switched it out so now if there's no gitTemplate it simply creates a fresh repository